src/entity/Menu.java

```diff
@@ -0,0 +1,74 @@
+package entity;
+
+import java.io.Serializable;
+
+public class Menu implements Serializable, Comparable<Menu> {
+    private Integer id;
+    private String name;
+    private String description;
+    private double price;
+    private String image;
+    private int parentId;
```

Comment:
Why did you need store parentId?

Reply:
Because we will need to find the menu by parentId
Ex: If we need to find the breakfast menu, we will search it by parentId which is 3 or another id